pythonx/jedi_vim.py

```diff
@@ -110,10 +110,14 @@ def echo_highlight(msg):
 
 try:
     import jedi
-except ImportError as e:
-    no_jedi_warning(str(e))
+except ImportError:
     jedi = None
-    jedi_import_error = str(e)
+    _, exc, tb = sys.exc_info()
+    while tb.tb_next:
+        tb = tb.tb_next
+    f_code = tb.tb_frame.f_code
+    exc_location = '%s:%d' % (f_code.co_filename, f_code.co_firstlineno)
```

Comment:
Why don't we just return the whole traceback with `import traceback`? Wouldn't that be nicer and less code?

Reply:
Good point, but not trivial.  I will improve this.